behave_django/management/commands/behave.py

```diff
@@ -165,7 +181,15 @@ def get_behave_args(self, argv=sys.argv):
         args, unknown = parser.parse_known_args(argv[2:])
 
         behave_args = []
-        for option in unknown:
+        while unknown:
+            option = unknown.pop(0)
+            # Remove runner since that was inserted directly into
+            # the run_behave()
+            if option == '--behave-runner':
+                # This option always has a value attached to it, we're also
+                # poping the next element to avoid errors.
+                unknown.pop(0)
+                continue
```

Comment:
This whole block is unclear (note the excessive use of comments, that supports this finding), though it really does a very specific thing (I believe): Removing *behave-django* specific options.

There are two ways to solve this problem:

1. Use a list variable, e.g. `django_behave_specific_options`, and `if option in django_behave_specific_options:`, or
2. Rename the option to not use a `--behave-` prefix at all (e.g. `--bdd-test-runner` or just `--test-runner`).

Both solutions make the use of comments unnecessary.

Personally, I'd prefer the latter approach as this would reduce the amount of code needed and retain the clean mental model of "`--behave-` comes from *behave*".